css/main.css

```diff
@@ -5,133 +5,138 @@
     color: #1b1b1b;
 }
 
-a {
+.no-underline {
     text-decoration: none;
 }
 
 header {
-    text-align: left;
-    padding-inline: 10%;
+    position: relative;
+    padding: 0 max(50px, 10%);
```

Comment:
Look, `max` has [bad compatibility](https://caniuse.com/css-math-functions) with old browsers, so we use combination of media queries and static values. Just keep fixed value there is. `50px`.